services/order_by.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+
+from django.contrib.auth.models import User
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(tickets: list[dict], username: str, date: datetime) -> list[dict]:
+    with transaction.atomic():
+        order = Order.objects.create(username=username, created_at=date)
```

Comment:
The `Order` model should not use a `username` field directly. Use a foreign key to the user model instead, and pass a `User` instance.